volatility3/framework/symbols/windows/crash64.json

```diff
@@ -65,25 +65,25 @@
           "offset": 40,
           "type": {
             "kind": "base",
-            "name": "unsigned long"
+            "name": "unsigned long long"
           }
         },
         "MachineImageType": {
-          "offset": 44,
+          "offset": 48,
```

Comment:
Ah, this was likely me taking from the 32-bit version to make a 64-bit version.  5:S  Sorry!